spec/controllers/spree/admin/products_controller_spec.rb

```diff
@@ -150,17 +150,31 @@
   end
 
   describe "updating" do
+    let(:producer) { create(:enterprise) }
+    let!(:product) { create(:simple_product, supplier: producer) }
+
+    before do
+      @request.env['HTTP_REFERER'] = 'http://test.com/'
```

Comment:
Why is this needed here?

Reply:
Yeah, it doesn't seem to belong here. :man_shrugging:

Reply:
must have been a copy paste error... removed in new commit.